src/main/typescript/wcardinal/ui/d-slider.ts

```diff
@@ -115,21 +114,27 @@ export abstract class DSlider<
 		};
 	}
 
-	protected newValue( options?: OPTIONS ): DSliderValue {
-		return new DSliderValue( options && options.value );
-	}
-
-	protected newMax(options?: OPTIONS): DSliderLabel {
-		const max = options && options.max;
-		return new DSliderLabel({
-			value: ( max && max.value ) ?? 1
-		});
-	}
+	protected newValues( options?: OPTIONS ): void {
+		// Get max / min value from options object
+		let max = options && options.max && options.max.value;
+		let min = options && options.min && options.min.value;
+		let value = options && options.value && options.value.value;
+		if (isNumber(max) && isNumber(min) && isNumber(value)) {
+			// Adjust if needed
+			max = Math.max( max, min );
+			value = Math.max( min, Math.min( max, value ) );
+		} else {
+			// Set as default
+			value = min = 0;
+			max = 1;
+		}
 
-	protected newMin( options?: OPTIONS ): DSliderLabel {
-		const min = options && options.min;
-		return new DSliderLabel({
-			value: ( min && min.value ) ?? 0
+		// Init min, max, value as default
+		this._min = new DSliderLabel({ value: min });
+		this._max = new DSliderLabel({ value: max });
+		this._value = new DSliderValue({ 
+			...{ value: value }, 
+			...(options?.value ?? {})
```

Comment:
Please make sure that the spread operation in object literals are supported in browsers which support WebGL.